Bottom: 1f785303da8568154fe26394154f78b371c60990
Top:    5dcb6450cc8c676d74e87ba8c35f589fa1b828f6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 00:30:12 +0100

Refresh of w-c-split-gathering-routines-f-0

---

diff --git a/src/draw.c b/src/draw.c
index 3b7fdab..ea5e9ce 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -734,12 +734,12 @@ draw_element_package (ELement)
   /* draw lines, arcs, text and pins */
   ELEMENTLINE_LOOP (Element);
   {
-    _draw_line (line);
+    DrawLineLowLevel (line);
   }
   END_LOOP;
   ARC_LOOP (Element);
   {
-    _draw_arc (arc);
+    DrawArcLowLevel (arc);
   }
   END_LOOP;
 }
